jcl/src/java.management/share/classes/com/ibm/java/lang/management/internal/LoggingMXBeanImpl.java

```diff
@@ -88,20 +126,26 @@ public static LoggingMXBeanImpl getInstance() {
 	@Override
 	public String getLoggerLevel(String loggerName) {
 		String result = null;
-		Logger logger = LogManager.getLogManager().getLogger(loggerName);
-
-		if (logger != null) {
-			// The named Logger exists. Now attempt to obtain its log level.
-			Level level = logger.getLevel();
-			if (level != null) {
-				result = level.getName();
-			} else {
-				// A null return from getLevel() means that the Logger
-				// is inheriting its log level from an ancestor. Return an
-				// empty string to the caller.
-				result = ""; //$NON-NLS-1$
+		
+		try {
+			Object logger = getLoggerFromName(loggerName);
+	
+			if (logger != null) {
+				// The named Logger exists. Now attempt to obtain its log level.
+				Object level = lo_getLevel.invoke(logger);
+				if (level != null) {
+					result = (String)le_getName.invoke(level);
+				} else {
+					// A null return from getLevel() means that the Logger
+					// is inheriting its log level from an ancestor. Return an
+					// empty string to the caller.
+					result = ""; //$NON-NLS-1$
+				}
 			}
+		} catch (Exception e) {
+			newInternalInvokeError(e);
```

Comment:
Should be `throw newInternalInvokeError(e)` as it makes it obvious from reading this code that execution can't continue past this point without needing to know the implementation of `newInternalInvokeError(e)`